setup.py

```diff
@@ -70,6 +70,7 @@
         "kaitaistruct>=0.7,<0.9",
         "ldap3>=2.5,<2.6",
         "passlib>=1.6.5, <1.8",
+        "ply>=3.4, <3.12",
```

Comment:
Did you actually test this with ply 3.4?
That release is pretty old (2011), so I think we can bump this to at least 3.6 (2015) or even 3.10 (2017)...